CONTRIBUTING.md

```diff
@@ -141,6 +141,33 @@ to email the patch to <gap@gap-system.org>.  You can refer to
 [this tutorial](https://burzalodowa.wordpress.com/2013/10/05/how-to-send-patches-with-git-send-email/)
 on how to do this.
 
+## Reviewing code contributions
+
+When reviewing code, you can use the following list as a checklist.
+
+### Before you even run the code (see [here](https://lornajane.net/posts/2015/code-reviews-before-you-even-run-the-code))
+
+* Is it clear what feature / fix the contribution adresses?
+* Does it relate to exactly one issue?
+* Do the changed files look sensible? Use `git log --stat` to discover
+possibly unintentional changes to files.
```

Comment:
indent this line to align with the list text above it

Reply:
Also, the above assumes that one has cloned the code locally. Which one can of course do -- but I assume many people will just review in the web interface, and there it is of course also possible to perform this check. So perhaps rephrase this to be neutral of the medium. E.g. 
> Does the list of changed files look sensible, or are there perhaps files with unintentional changes? On GitHub, do XYZ to find out; if you have clone the PR, you can use `git log --stat`.